packages/perseus/src/widgets/image.tsx

```diff
@@ -97,37 +97,20 @@ class ImageWidget extends React.Component<Props> {
 
         const backgroundImage = this.props.backgroundImage;
 
+        if (this.props.caption === this.props.alt) {
+            alt = "";
+        } else {
+            alt = this.props.alt;
+        }
```

Comment:
Nit: I'm thinking that we can now write this as a ternary:
`const alt = this.props.caption === this.props.alt ? "" : this.props.alt`